src/parselmouth/Pitch.cpp

```diff
@@ -239,6 +242,37 @@ PRAAT_CLASS_BINDING(Pitch) {
 
 	// TODO To PitchTier: depends on PitchTier
 
+	def(
+	        "to_point_process",
+	        [](Pitch self, Sound sound, std::string method, bool include_maxima, bool include_minima) {
+		        if (sound) {
+			        if (method == "cc")
```

Comment:
So, yes, you know what I'm going to say about this one. I appreciate the suggestion and it has made me think about things, but I would propose to now go for consistency with e.g. `Sound.to_pitch` and `Sound.to_harmonicity`, and turn this into an enum.

Separately, we can explore the options of having lowercase strings for all enum, or something similar. The advantage here is that we can consider all existing enums and not create confusion between "here it's an enum or a SCREAMING_SNAKE_CASE string" vs. "oh, here it's a snake_case string".

Reply:
One of the arguments for enums is that the interface describes itself (through the type), rather than that you have to look at the docstrings (which are missing in most cases). So I do see that your thorough documentation makes the use of strings a lot easier!

Reply:
By the way, if you agree, I will make the change myself! Not expecting you to do that.

But I don't wish to push this through too dictatorially; the fact that you are expecting lowercase strings is an interesting point and I must admit it kind of makes sense from a "Pythonic" perspective. I'd just like to 1) study the possibilities and (dis)advantages, 2) do something consistently across all things, and not just for this new class.

Reply:
Enums are fine with me as they are pretty much transparent to me (and most other end users). I'll look into case-insensitive enum-string conversion